src/styles/components/_modal.scss

```diff
@@ -0,0 +1,76 @@
+.bsds-modal
+{
+  box-shadow: $box-shadow-primary;
+  border: 0;
+  padding: space(4x);
+  background: $neutral-100;
+  max-width: 90vw;
+  min-width: min(600px, 100vw);
+
+  &::backdrop
+  {
+    background: rgba($neutral-00, 0.6);
+  }
+
+
+  .bsds-modal-header
+  {
+    display: grid;
+    grid-template-columns: 1fr auto;
+    grid-template-areas:
+      "logo close"
+      "title close";
+    align-items: start;
+    gap: space(3x);
+  }
+
+  .bsds-modal-logo
+  {
+    grid-area: logo;
+    margin-top: 0;
+    width: auto;
+    height: space(6x);
+
+    & + .bsds-modal-title{
+      grid-row: span 1;
+    }
+  }
+
+  .bsds-modal-title
+  {
+    grid-area: title;
+    grid-row: span 2;
+    align-self: center;
+    margin-top: 0;
+  }
+
+  .bsds-modal-close
+  {
+    grid-area: close;
+  }
+
+  .bsds-modal-body
+  {
+    margin-top: space(4x);
+  }
+
+  .bsds-modal-footer
+  {
+    display: flex;
+    flex-wrap: wrap;
+    gap: space(2x);
+    margin-top: space(8x);
+
+    a.bsds-button-assertive
```

Comment:
Anything we should look at modifying in the mixins to avoid having to add this?

Reply:
This is because its competing with link styles, so its an override. We would need to up the importance of button styles if we want to say that button always beats out link styles.

Reply:
Mm I'm not sure we can say that confidently in all cases. Let's keep an eye out for button/link element+class overrides so we can make that call eventually.